extension/package.json

```diff
@@ -6,7 +6,8 @@
     "postinstall": "npm link ../signer/web && npm link ../components",
     "start": "npm run devBuild -- -w",
     "devBuild": "webpack --progress --colors",
-    "build": "webpack --progress --colors --bail",
+    "prebuild": "npm run clean",
+    "build": "NODE_ENV=production webpack --progress --colors --bail",
```

Comment:
This doesn't work on all platforms - please create a separate `build` step for yourself.
Also please remove `prebuild` - will break some caching mechanism on travis/CI.